svg-min/medium.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect x="0" fill="none" width="24" height="24"/><g><style>.st0{fill:#231F20;}</style><path class="st0" d="M20.9 7.2l-5.4-2.7h-.1-.1c-.2 0-.4.1-.5.3l-3.1 5.1 3.9 6.4 5.3-9.1c.1 0 .1 0 0 0zM9.9 8.6v5.8l5.1 2.5zM15.8 17.3l4.2 2.1c.6.3 1 .1 1-.4V8.8l-5.2 8.5zM9 7.2L3.7 4.5c-.1 0-.2-.1-.3-.1-.2.1-.4.2-.4.5v11.4c0 .3.2.7.5.8l4.7 2.3c.1.1.2.1.3.1.3.1.5-.1.5-.5V7.2z"/></g></svg>
```

Comment:
We should remove this `id="Layer_1"` and anywhere else it affects (generated files included).